server.go

```diff
@@ -1110,11 +1110,23 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 		ScidCloser:              scidCloserMan,
 	}, nodeKeyDesc)
 
+	selfVertex := route.Vertex(nodeKeyDesc.PubKey.SerializeCompressed())
 	//nolint:lll
 	s.localChanMgr = &localchans.Manager{
-		SelfPub:                   nodeKeyDesc.PubKey,
-		DefaultRoutingPolicy:      cc.RoutingPolicy,
-		ForAllOutgoingChannels:    s.graphBuilder.ForAllOutgoingChannels,
+		SelfPub:              nodeKeyDesc.PubKey,
+		DefaultRoutingPolicy: cc.RoutingPolicy,
+		ForAllOutgoingChannels: func(cb func(kvdb.RTx,
```

Comment:
Add a comment here or a note, that the user must catch the `edge = nil` in his callback

Reply:
Done